File: sort/python3/bubble_sort.py
size = int(input("Enter size of array: "))
print(size)

array = []

print("Enter the array elements: ")
element = input().split(' ')
for i in range(0, size):
    array.append(int(element[i]))

for j in range(0, size):
    for l in range(size-1, 0, -1):
        if array[l] < array[l-1]:
            array[l], array[l-1] = array[l-1], array[l]
print(array)

File: sort/cpp/bubble_sort.cpp
#include <iostream>
#include <algorithm> // for swap
#include "helper.h" // printArray()
#include "sort_functions.h"
using namespace std;

int main()
{   
    std::ios::sync_with_stdio(false); // for faster input and output using cin and cout
    int size;
    cout << "Enter the size of your array: ";
    cin >> size;
    int arr[size];
    cout << "Enter your array elements\n";
    for(int i=0; i<size; i++)
    {   
        cin >> arr[i];
    }
    
   bubble_sort(arr, size);
   cout << "Array after sorting is: ";
   printArray(arr, size);	

   return 0;
}
